app/assets/stylesheets/daypicker.css

```diff
@@ -0,0 +1,133 @@
+/* DayPicker styles */
```

Comment:
Yes, I mentioned that in the commit, but the rails-assets bower package does not seem to bring in the styes.